website/src/components/NavBar.js

```diff
@@ -22,7 +22,7 @@ import {
     Logout
 } from '@mui/icons-material';
 import MenuIcon from '@mui/icons-material/Menu';
-import api from '../utils/api';
+import api from '../utils/apiv2';
```

Comment:
If you're using apiv2, to differentiate, use the name apiv2 instead

Reply:
In that case, we need to change `buckets.js` as well. Should this be in a separate PR?

Reply:
Yeah we can do that in a separate PR.  Just make a p0 ticket to do this